server/src/messageHandler.mjs

```diff
@@ -35,7 +36,7 @@ function emit(id, result, msg, ws) {
       result: result
     };
     const eventMessage = JSON.stringify(oEventMessage);
-    ws.send(eventMessage);
+    ws.send(eventMessage);    
```

Comment:
Looks like some stray spaces / tab at the end of this line.

Reply:
removed stray white space at line # 39